api/portainer.go

```diff
@@ -46,12 +46,13 @@ type (
 
 	// LDAPSettings represents the settings used to connect to a LDAP server.
 	LDAPSettings struct {
```

Comment:
For the migration to work, you'll need to update the database version constant in this file.